ui/ozone/platform/starboard/surface_factory_starboard.cc

```diff
@@ -16,8 +16,24 @@
 
 namespace ui {
 
-SurfaceFactoryStarboard::SurfaceFactoryStarboard() {}
+SurfaceFactoryStarboard::SurfaceFactoryStarboard() : egl_implementation_() {}
```

Comment:
`: egl_implementation_() {}` is unnecessary since `egl_implementation_` is a fully owned object with a no-arg constructor so instead do just `= default;`. Same for the dtor in l.21.